src/cozy/wrappers/sharing/sharing.jsx

```diff
@@ -12,12 +13,36 @@ const Sharing = ({
     onShared
 }) => {
   const [showShareModal, setShowShareModal] = useState(false)
+  const [hasRecipientsToBeConfirmed, setHasRecipientsToBeConfirmed] = useState(false)
+
+  const checkRecipientsToBeConfirmed = async () => {
+    const result = await confirmationMethods.getRecipientsToBeConfirmed()
+
+    console.log({result})

```

Comment:
to remove

Reply:
Good catch. [Removed](https://github.com/cozy/cozy-pass-web/compare/e0ad233bdb2a2370e2f6c104c3437a0fd1e29b17..b7df4891237b505683409078761ba8c5c49a3428)